### PR TITLE
{bio}[GCCcore/11.3.0] CapnProto v0.10.2

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jfgrimm** [comment]
@lexming we already have version 0.9.1 with this toolchain (#15043); is this needed as a dependency for anything else? otherwise, perhaps move it to GCCcore11.3.0?

File: easybuild/easyconfigs/c/CapnProto/CapnProto-0.10.2-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+source_urls = ['https://capnproto.org/']
+sources = ['capnproto-c++-%(version)s.tar.gz']
+checksums = ['7cd91a244cb330cda5b41f4904f94b61f39ba112835b31fa8c3764cedbed819f']
+
+builddependencies = [('binutils', '2.37')]
+
+sanity_check_paths = {
+    'files': ['bin/capnp', 'bin/capnpc', 'bin/capnpc-c++', 'bin/capnpc-capnp'],
+    'dirs': ['include/capnp', 'include/kj', 'lib'],
+}
```
Comment (jfgrimm) [inline easybuild/easyconfigs/c/CapnProto/CapnProto-0.10.2-GCCcore-11.2.0.eb:29]:
```suggestion
local_bins = ['capnp', 'capnpc', 'capnpc-c++', 'capnpc-capnp']

sanity_check_paths = {
    'files': ['bin/%s' % x for x in local_bins],
    'dirs': ['include/capnp', 'include/kj', 'lib'],
}

sanity_check_commands = ["%s --help" % x for x in local_bins]
```

**Comment by lexming** [comment]
@jfgrimm good point, it will be more useful in GCCcore/11.3.0

**Comment by jfgrimm** [comment]
@boegelbot please test @ generoso

**Comment by jfgrimm** [comment]
@boegelbot please test @ jsc-zen2

**Review by jfgrimm — APPROVED** [review]
LGTM

**Pull request merged**